extensions/components/com_redcore/site/layouts/translation/fields/state.php

```diff
@@ -0,0 +1,53 @@
+<?php
```

Comment:
That'd certainly be a preferable approach. I was kind of operating under the assumption, that frontend (site) and backend (admin) was completely separated from each other, but it seems like that's not the case after all.